pkg/sql/logictest/testdata/logic_test/interleaved

```diff
@@ -304,3 +304,17 @@ interdb                               CREATE TABLE interdb (
                                       CONSTRAINT "primary" PRIMARY KEY (x ASC),
                                       FAMILY "primary" (x)
 ) INTERLEAVE IN PARENT other.foo (x)
+
+statement ok
+create table t1 (id1 int primary key, id2 int, id3 int);
```

Comment:
nit: please capitalize the SQL keywords in tests. This helps us grepping for them later.